src/gui/folderman.cpp

```diff
@@ -837,6 +845,22 @@ bool FolderMan::pushNotificationsFilesReady(Account *account)
     return pushFilesAvailable && pushNotifications && pushNotifications->isReady();
 }
 
+bool FolderMan::switchToVfsIsNeeded(const FolderDefinition &folderDefinition) const
+{
+    auto result = false;
+    if (ENFORCE_VIRTUAL_FILES_SYNC_FOLDER) {
+        const auto bestVfsPlugin = bestAvailableVfsMode();
```

Comment:
Nit: You can remove the variable. It's only used in one place and the name is already short and speaking in my oppinion.